src/bundle/Resources/views/content/content_preview.html.twig

```diff
@@ -42,7 +42,7 @@
                 <div class="ez-preview__item ez-preview__item--siteaccess">
                     <select class="form-control">
                         {% for previewSiteaccess in siteaccesses %}
-                            <option value="{{ path('_ezpublishPreviewContent', {'contentId': content.id, 'versionNo': versionNo, 'language': language_code, 'siteAccessName': previewSiteaccess}) }}">{{ previewSiteaccess }}</option>
+                            <option value="{{ path('_ezpublishPreviewContent', {'contentId': content.id, 'versionNo': versionNo, 'language': language_code, 'siteAccessName': previewSiteaccess}) }}">{{ siteaccess_label(previewSiteaccess) ~ ' (' ~ previewSiteaccess ~ ')' }}</option>
```

Comment:
@lserwatka @webhdx I would use just: `previewSiteaccess|trans({}, 'ezplatform_siteaccess')` instead of TWIG function for now in that case. Better to implement twig helper directly in new Core bundle to avoid BC break instead of having two or one of them deprecated or something.

Reply:
Indeed a TWIG helper will be very useful for frontend developer. Definitely something to add to core bundle.
ping @bdunogier

Reply:
Well, as I told during demo this will go into new separate package.
@lserwatka